norne/INCLUDE/BC0407_RESTART.SCH

```diff
@@ -3148,9 +3148,9 @@ WCONHIST
     'E-3AH'      'OPEN'      'RESV'   1609.046    178.782 626599.537  5* /
      'E-2H'      'OPEN'      'RESV'   5792.532      0.000 949276.709  5* /
     'E-4AH'      'OPEN'      'RESV'   1485.125      0.000 141254.548  5* /
+/
 
-
-WCONINJ
+WCONINJE
```

Comment:
Please check this closely as that change could change the semantics of the input deck.  `WCONINJ` is the less flexible precursor to `WCONINJE`.

Reply:
Yes but: 

1. We do not support `WCONINJ` at all - the json keyword is just a placeholder.
2. All other ocurences in the deck are `WCONINJE`.

So - I am very confident this is a typo.